### ISSUE TITLE
Discovered / Fixed issue with Conjure Xivilai Lord Spell

### ISSUE THREAD

**Issue by smadam813** [ForgottenGlory/Living-Skyrim-2#329]
I am on the latest version of LS2.

There was a patch that was being applied which was taking care of an issue of a perk not executing correctly.  I decompiled the script and it has the following logic.
```
if game.GetPlayer().GetActorValue("magicka") <= 1 as Float
   debug.Notification("Insufficient Magicka")
   togglespellglobal.Mod(-1.00000)
   game.GetPlayer().RemoveSpell(togglespellnegative)
```
As you can see, this will cause problems with summoning a Xivilai since it depletes whatever amount of magicka you have.  Then once it depletes, it subtracts from your magicka pool.  hense, you are now in the negative.  The easier way to fix this script is to simply delete it and override the Spell form id's with one that has the Ordinator master.

Solution TLDR:
- Xivilai Lord costs all of your Magicka as a magical effect, then your magicka is reduced by x amount due to how Sustained Magic works which puts you into the negative.  Game no like this.
- Remove togglespellApocalypse script from Sustained Magic Patches as it makes it impossible to cast Xivilai Lord
- Merge https://www.nexusmods.com/skyrimspecialedition/mods/26450?tab=files (Sustained Magic - Apocalypse Ordinator Patch) into LS2 Patches OR add Ordinator as master to Sustained Magic - Apocalypse.esp
- I suppose 3rd option would be to rewrite that script to make an exception for this specific spell, however that seems like a waste of time since less scripts running == better.

Any questions you can hit me up on discord: Torhild#8380

**Comment by smadam813** [comment]
After further testing, I don't think any patching is actually required, just removal of that script file.  I didn't realize at first, but apparently that patch was provided via ForgottenGlory lol.

**Comment by ForgottenGlory** [comment]
Fixed 2.2.0
